src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
+        machines[0] = new Truck();
+        machines[1] = new Bulldozer();
+        machines[2] = new Excavator();
+        for (Machine machine : machines) {
+            machine.doWork();
```

Comment:
The messages printed in the `doWork` and `stopWork` methods should be more informative. They should indicate what type of `Machine` is working or stopping. Make sure the messages in those methods are updated accordingly.